### PR TITLE
Typo to js framework pages

### LINKED ISSUE #4910: Fix backend Javascript callout
https://www.highlight.io/docs/getting-started/backend-sdk/js/overview
Screenshot 2023-04-10 at 8 52 15 AM

Should say "If there's a Javascript Framework that's missing"

### PR DESCRIPTION
**Pull request by jay-khatri**
## Summary
- Resolves #4910 
- Adds a `MissingFrameworkCopy` component to be reused across the docs

## How did you test this change?
Docs only.

## Are there any deployment considerations?
n/a

### CODE REVIEW

File: docs-content/getting-started/4_backend-sdk/js/1_overview.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ updatedAt: 2022-04-01T19:52:59.000Z
 ---
 
 ```hint
-If there's a Go Framework that's missing, feel free to [create an issue](https://github.com/highlight/highlight/issues/new?assignees=&labels=external+bug+%2F+request&template=feature_request.md&title=) or message us on [discord](https://highlight.io/community).
+If there's a framework that's missing, feel free to [create an issue](https://github.com/highlight/highlight/issues/new?assignees=&labels=external+bug+%2F+request&template=feature_request.md&title=) or message us on [discord](https://highlight.io/community).
```
[the remark this answers is not in the document]
Comment (jay-khatri) [reply docs-content/getting-started/4_backend-sdk/js/1_overview.md:10]: Prefer to make it more generic so I can copy/paste for new ones.

[the remark this answers is not in the document]
Comment (jay-khatri) [reply docs-content/getting-started/4_backend-sdk/js/1_overview.md:10]: ok, will update.

[the remark this answers is not in the document]
Comment (jay-khatri) [reply docs-content/getting-started/4_backend-sdk/js/1_overview.md]: added a component to share across all these docs.

File: highlight.io/pages/docs/[[...doc]].tsx [hunk 2]
```diff
@@ -1346,4 +1347,12 @@ const QuickStart = (content: { content: QuickStartContent }) => {
 	)
 }
 
+const MissingFrameworkCopy = () => {
```
Comment (et) [inline highlight.io/pages/docs/[[...doc]].tsx:1350]: 👏

**Review by et — APPROVED** [review]

**Pull request merged**